### PR TITLE
Update fetch polyfill

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3168

We're seeing some failures in IE11 when posting forms and I want to make sure we're not hitting bugs because we use an older version of the fetch polyfill. This change affects primarily IE11, but also some older iOS and Android devices.

The only breaking change listed for the fetch polyfill in 2.0 is using `getAll()` with the Headers object, which I can't find any instances of us doing.

I also removed some instances of `mode: 'cors'` because that's the default and is unnecessary.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
     "libxmljs": "^0.18.0",
     "lodash": "^4.5.1",
     "lodash-deep": "^2.0.0",
-    "mapbox": "^1.0.0-beta5",
```
Comment (jbalboni) [inline package.json]: We're already using beta7, this just cleans up the yarn.lock file a little (it had multiple versions of whatwg-fetch before I changed this).

**Review by cvalarida — APPROVED** [review]
It'd be nice to have some cross-browser testing right about now, eh? Hmm... 🤔 Well, it looks good to me.

**Review by U-DON — APPROVED** [review]

**Review by bshyong — APPROVED** [review]
LGTM

**Comment by ayaleloehr** [comment]
@cvalarida if you want to use SauceLabs, here are the credentials: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Cross-Platform%20Testing/cross-platform-testing-plan.md

Not automated (yet?), but at least it's an easier way of using IE than I had before!

**Pull request merged**